sql/firefox_kpi_dashboard_v1.sql

```diff
@@ -73,22 +68,37 @@ function sum_buckets_with_ci(counts_per_bucket) {
         "nbuckets": n,
     };
 }
-
 return sum_buckets_with_ci(a);
 """;
-  --
+--
 WITH
   forecast_base AS (
   SELECT
     *
   FROM
     `moz-fx-data-derived-datasets.analysis.growth_dashboard_forecasts_current` ),
   --
+  desktop_min_forecast_date AS (
+  SELECT
+    AS VALUE MIN(`date`)
+  FROM
+    forecast_base
+  WHERE
+    datasource = 'desktop_global'
+    AND type = 'forecast' ),
+  --
   desktop_base AS (
   SELECT
     *
   FROM
-    `moz-fx-data-derived-datasets.telemetry.firefox_desktop_exact_mau28_by_dimensions_v1` ),
+    `moz-fx-data-derived-datasets.telemetry.firefox_desktop_exact_mau28_by_dimensions_v1`
+  WHERE
+    -- Temporarily freeze desktop data due to data deletion enacted on 2019-05-16.
+    submission_date < (
+    SELECT
+      *
+    FROM
+      desktop_min_forecast_date)),
```

Comment:
This code looks to be from an older version of this table. The changes shown are reverting to an older revision of the logic. Can you check the latest master and make sure this file matches?

Reply:
Right, I should have rebased this branch before moving files to other directories. Anyway, I copied all the SQL files from the current `master` and moved them to `templates/`. So everything should be up to date now